packages/grafana-data/src/themes/createTheme.test.ts

```diff
@@ -0,0 +1,101 @@
+import { createTheme } from './createTheme';
+
+describe('createTheme', () => {
+  it('create default theme', () => {
+    const theme = createTheme();
+    expect(theme).toMatchInlineSnapshot(`
+      Object {
+        "breakpoints": Object {
+          "down": [Function],
+          "keys": Array [
+            "xs",
+            "sm",
+            "md",
+            "lg",
+            "xl",
+            "xxl",
+          ],
+          "unit": "px",
+          "up": [Function],
+          "values": Object {
+            "lg": 992,
+            "md": 769,
+            "sm": 544,
+            "xl": 1200,
+            "xs": 0,
+            "xxl": 1440,
+          },
+        },
+        "colors": Object {
+          "background": Object {
+            "layer0": "#0b0c0e",
+            "layer1": "#141619",
+            "layer2": "#202226",
+            "layer3": "#2c3235",
+          },
+          "border": Object {
+            "b1": "#141619",
+            "b2": "#202226",
+            "b3": "#2c3235",
+          },
+          "contrastThreshold": 3,
+          "error": Object {
+            "contrastText": "#e02f44",
+            "dark": "darken(color.main, tonalOffset)",
```

Comment:
is this correct?

Reply:
Ignore ^^ .  I see now in the code :)

Reply:
No just a placeholder as we have no color manipulation stuff in data yet